### ISSUE TITLE
⚠️  Looking for maintainers for reactjs-popup library

### ISSUE THREAD

**Issue by yjose** [yjose/reactjs-popup#332]
Hello fellow developers,

Due to personal and professional commitments, I am unable to continue maintaining the library at this time. I am reaching out to the community to see if there are any interested parties who would like to take on the role of maintaining this library.

The reactjs-popup library is a highly useful tool for creating modals, tooltips, and other types of pop-ups in React projects. It has a solid user base and is widely used.

As a maintainer, you would be responsible for:

* Responding to issues and pull requests
* Reviewing and merging pull requests
* Keeping the library up-to-date with the latest versions of React and other dependencies
* Ensuring that the library is stable and working correctly

If you are interested in becoming a maintainer for this library, please let me know by commenting on this issue. I will then provide you with the necessary permissions and guidance to get started.

**Comment by amila-t-kumarasekara** [comment]
I would like to be a maintainer of this library.I don't wanna outdated this library.

**Comment by shouvSarker** [comment]
Happy to be a maintainer. I am using this in a couple of places so probably have some vested interest in ensuring the project remains up-to-date.

**Comment by mithenji** [comment]
🙋 I want to participate in the maintenance and upgrade of this library, and want to adapt it to React v18.

**Comment by mistahoward** [comment]
I'm interested. We use this component in multiple projects in our organization. Would love to be able to contribute to it.

**Comment by orpic** [comment]
How much of a skill level would you require for a maintainer?
Also how much time you had to spend on maintaining,  on a weekly basis for the same ( when you were actively maintaining )?

**Comment by jeevan-aloor** [comment]
I'm interested. Would love to be able to contribute to it. and  i used this in my multiple project.

**Comment by carloskim123** [comment]
Yeah it seems like a good idea, count me in

**Comment by kostia-lev** [comment]
I am interested! I used this lib in my project.

**Comment by BataevDaniil** [comment]
I want be maintainer

**Comment by jeevan-aloor** [comment]
I'm interested in contributing.

**Comment by AhmedMuhammedElsaid** [comment]
I'm Interested also

**Comment by junzero741** [comment]
I'm interested! I like this project.

**Comment by singhkaram** [comment]
I am interested! I used this lib in my project.

**Comment by vrinceanuv** [comment]
Count me in as well :)

**Comment by Tosinpeter** [comment]
Interested , Count me in as well

**Comment by NahidAhmed47** [comment]
I want to contribute in this package. I am a experience Frontend developer. Count me as well. 
Thanks

**Comment by SegunDatamynt** [comment]
Hi, I would love to be a maintainer , how do we proceed.

**Comment by techiechandan** [comment]
I would like to work as a maintainer for this library.

**Comment by yjose** [comment]
Sorry guys, i was a bit busy and its very hard to select people, please if you really interested you can reserve a [spot here](https://calendly.com/yjose/meet) for a quick call

**Comment by 6km** [comment]
Hi @yjose 

Is there a way to get in contact with you?

**Comment by stonetwig** [comment]
Hello @yjose,

I am a dev from Sweden with more than a decade long experience in web dev. I am utilizing this library in many projects and thus don't want it to be inactive, fall into bad hands or something like that. I know that this ticket was opened for quite some time but I haven't seen it until now.

If you haven't found someone I have the time and will to take over and keep it maintained and updated. Tried to reserve a spot but it seems like the website you linked doesn't really load?

Best regards,
stonetwig

**Comment by jimmy-chiang** [comment]
Is there any updates with this library? Just realize that the document website is down...

**Comment by SegunDatamynt** [comment]
There are no feedbacks from him, he stated he has been very busy. Perhaps we can all come together and build another that would be properly maintained in this regard.What do you guys think?  Sent from my iPhoneOn 11 Nov 2024, at 9:09 AM, Jimmy Chiang ***@***.***> wrote:﻿
Is there any updates with this library? Just realize that the document website is down...

**Comment by vrinceanuvalentin** [comment]
@SegunDatamynt he left a link to set up a short call with him if you want to become a maintainer. Here it is: https://calendly.com/yjose/meet

**Comment by jimmy-chiang** [comment]
> There are no feedbacks from him, he stated he has been very busy. Perhaps we can all come together and build another that would be properly maintained in this regard.What do you guys think?  Sent from my iPhoneOn 11 Nov 2024, at 9:09 AM, Jimmy Chiang ***@***.***> wrote:﻿ Is there any updates with this library? Just realize that the document website is down... —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

@SegunDatamynt Agreed! I tried shadcn dialog today, and the experience wasn't as smooth as this library. I really think this library needs to continue to exist because of it's great development experience.

**Comment by SegunDatamynt** [comment]
Yea we scheduled a call sometimes ago but he was no show. Sent from my iPhoneOn 11 Nov 2024, at 9:20 AM, vrinceanuvalentin ***@***.***> wrote:﻿
@SegunDatamynt he left a link to set up a short call with him if you want to become a maintainer. Here it is: https://calendly.com/yjose/meet

**Comment by SegunDatamynt** [comment]
Great, let’s create a group for interested parties and work on it together. Sent from my iPhoneOn 11 Nov 2024, at 9:24 AM, Jimmy Chiang ***@***.***> wrote:﻿

There are no feedbacks from him, he stated he has been very busy. Perhaps we can all come together and build another that would be properly maintained in this regard.What do you guys think?  Sent from my iPhoneOn 11 Nov 2024, at 9:09 AM, Jimmy Chiang @.> wrote:﻿ Is there any updates with this library? Just realize that the document website is down... —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: @.>

@SegunDatamynt Agreed! I tried shadcn dialog today, and the experience wasn't as smooth as this library. I really think this library needs to continue to exist because of it's great development experience.

**Comment by SegunDatamynt** [comment]
Hi Guys, we want to start working on this repository now and would love to discuss issues to fix first. 
Please join us here to contribute to this :
Welcome to reactjs-popup Discussions! · yjose reactjs-popup · Discussion #356
f

| 
| 
| 
|  |  |

 |

 |
| 
|  | 
Welcome to reactjs-popup Discussions! · yjose reactjs-popup · Discussion...

� Welcome! We’re using Discussions as a place to connect with other members of our community. We hope that you:...
 |

 |

 |


First we do some fixes, then we do some upgrades. Let us work together on this, join the discussion.

    On Monday, November 11, 2024 at 12:44:11 PM GMT+1, Oluwasegun Aroyehun ***@***.***> wrote:  
 
 Great, let’s create a group for interested parties and work on it together. 
Sent from my iPhone

On 11 Nov 2024, at 9:24 AM, Jimmy Chiang ***@***.***> wrote:



﻿



There are no feedbacks from him, he stated he has been very busy. Perhaps we can all come together and build another that would be properly maintained in this regard.What do you guys think?  Sent from my iPhoneOn 11 Nov 2024, at 9:09 AM, Jimmy Chiang @.> wrote:﻿ Is there any updates with this library? Just realize that the document website is down... —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: @.>


@SegunDatamynt Agreed! I tried shadcn dialog today, and the experience wasn't as smooth as this library. I really think this library needs to continue to exist because of it's great development experience.

**Comment by maniac787** [comment]
I would like to:
Keeping the library up-to-date with the latest versions of React and other dependencies

**Comment by Gracecom1** [comment]
"Hi,

I saw your open maintainer request for reactjs-popup and the note that personal and professional commitments prevent you from continuing maintenance. I’m interested in preserving the established package and user base.

I’m interested in taking long-term responsibility for the project. My initial focus would be:

- update compatibility with current React and TypeScript
- strengthen accessibility, tests, dependency management, and releases
- keep the existing free package available while preparing a transparent maintenance roadmap

Could we continue this conversation privately by email or DM? I would prefer not to discuss transfer or commercial terms in a public issue. You can reach me at contact@paradigmjp.com.

Best regards,

Tomohiro Ha
Paradigm LLC
contact@paradigmjp.com
https://paradigmjp.com"

**Comment by vrinceanuvalentin** [comment]
Shall we fork this in order to bring it to today's standards?
